components/Tabs.tsx

```diff
@@ -48,4 +48,4 @@ const Tabs = ({ children }) => {
   );
 };
 
-export default Tabs;
+export default Tabs;
```

Comment:
all imports that were previously using the `Tab` component should be updated to `Tabs`